doc/server-vs-broker.md

```diff
@@ -0,0 +1,22 @@
+## Why is there a need for the broker?
+The KUKSA.val server started as a C++ VISS implementation, that later evolved and gained gRPC support. KUKSA.val databroker was a similar project trying to build a modern rust server for VSS data using a binary efficient gRPC protocol. Currently both projects are being joined, and will support the same GRPC API and Python library so you can exchange feeders and apps.

```

Comment:
We should better align on how to refer to databroker in text. Here we use "KUKSA.val databroker", in other places in this PR "gRPC broker", "data broker" or "data Broker".